### ISSUE TITLE
Unable to Revert to Bing Data

### ISSUE THREAD

**Issue by bibbster12** [derekhe/msfs2020-map-enhancement#563]
**Describe the bug**

Hello, I'm having an issue with reverting back to Bing maps for when not using the app.  It's stuck with data that for this specific area (KLNC airport).  The data appears to be the same source Apple map and Bing Maps (latest).  This issue is only in areas I've previously used the app in.  

Things I have tried (not in order):

-Uninstall app (via add/remove programs) / reinstalled app
-Delete all app related folders in Program Files and ALL appdata folders 
-Always ran the app with MSFS CACHE turn off
-Cleared Cache in SIM and in app
-Restarted Computer
-Cleared the host file in sys32 folder (as described in manual)
-Start app, _then_ open sim as instructed 

Additional information:

Other maps load up fine, no problems, but no matter what I try (as explained above), I cannot clear this Apple/Bing (latest) data cache that is coming from somewhere.  If I load into an area I never loaded into before (with the app off or uninstalled), default Bing maps works fine.  The issue is only in areas where I have used the app in before.  

Thank you.  I look forward to your observations.

**Map Enhancement version:**
- 7.5.3

**Device ID**
- CGQFAB2SB29DPJXPBMBC6RW4ZV4N9567G999J82HH8JHC2N2V540

**Comment by bibbster12** [comment]
Here to bump the issue as it has been 3 weeks. Any answer to this issue? I would like invest in the yearly license, but want to know the resolution to this issue.

**Comment by derekhe** [comment]
You can try : https://github.com/derekhe/msfs2020-map-enhancement/wiki/Manual-EN#615-after-exiting-and-re-entering-the-game-the-game-loads-very-slowly-and-displays-the-default-image

**Comment by marioedani** [comment]
Hi, I have the same problem. I tested this program to replace bing maps with google, but in my zone Bing maps have more realistic colors. So I wanted to revert to the default Bing maps, but I can't. Google maps textures are shown mixed with bing ones. I cleared the cache but the problem is still there.

How can I delete google maps textures and revert back to the default bing maps textures?

Thank you
